plugins/inputs/mysql/README.md

```diff
@@ -196,7 +202,9 @@ measurement name.
 * Slave status - metrics from `SHOW SLAVE STATUS` the metrics are gathered when
 the single-source replication is on. If the multi-source replication is set,
 then everything works differently, this metric does not work with multi-source
-replication.
+replication, unless you set `gather_all_slave_channels = true`. For MariaDB,
+`mariadb_dialect = true` should be set to address the field names and commands
```

Comment:
Does percona have it's own dialect as well? Maybe this would be better suited as an enum, something like `sql_dialect = "mariadb"` where the default/blank value is `"mysql"`, for extensibility.

Reply:
No, Percona uses MySQL's one.